core/downloadfromdcthread.cpp

```diff
@@ -184,8 +184,10 @@ void fill_computer_list()
 		descriptorLookup[QString(vendor) + QString(product)] = descriptor;
 	}
 	dc_iterator_free(iterator);
-	Q_FOREACH (QString vendor, vendorList)
-		qSort(productList[vendor]);
+	Q_FOREACH (QString vendor, vendorList) {
```

Comment:
Huh? Now I'm confused - weren't all Q_FOREACH replaced by range-based for?

Reply:
No, not all. As the commit message explains: only those that can easily be made const. Here I would need std::as_const, qAsConst or a "const auto l = vendorList; for (auto v: l)".

Reply:
That's horrible. Point taken.